src/ocamlorg_frontend/pages/package_overview.eml

```diff
@@ -153,7 +153,7 @@ let num_of_dependencies_to_show = 100 in
 let render_collapsible_dependencies section =
   let is_collapsible = section.collapsible && List.length section.items > num_of_dependencies_to_show in
   <div <%s! if is_collapsible then "x-data='{ open: false }'" else "" %>>
-    <div class="border-l-4 px-4 -ml-4 mb-6 mt-12 border-transparent border-l-body-400 rounded border-t-2 border-b-2 border-r-2">
+    <div class="border-l-4 border-l-gray-800 border-b-2 border-t-2 border-r-2 rounded border-transparent px-4 -ml-4 mb-6 mt-12">
```

Comment:
```suggestion
    <div class="border-l-4 border-l-[#575959] border-b-2 border-t-2 border-r-2 rounded border-transparent px-4 -ml-4 mb-6 mt-12">
```

In this case, since `gray-800` quite a bit different to the color in the Figma document, I would go with the explicit color value from the Figma document, for now.

There will be color variables coming from Figma for the dark mode project, but these aren't entirely ready yet, so we have to go with something that resembles the design as best as we can.